core/iwasm/common/wasm_runtime_common.c

```diff
@@ -507,7 +507,7 @@ wasm_runtime_is_xip_file(const uint8 *buf, uint32 size)
 
 #if (WASM_ENABLE_THREAD_MGR != 0) && (WASM_ENABLE_DEBUG_INTERP != 0)
 uint32
-wasm_runtime_start_debug_instance(WASMExecEnv *exec_env)
+wasm_runtime_start_debug_instance(WASMExecEnv *exec_env, int32_t port)
```

Comment:
This is a public API and has been used in some other codes.

Had better remain this API unchanged and add a new API `wasm_runtime_start_debug_instance_with_port`, then `wasm_runtime_start_debug_instance` can call `wasm_runtime_start_debug_instance_with_port` with port = -1

Reply:
it makes sense. thank you. done.